bindata/bootkube/bootstrap-manifests/kube-controller-manager-pod.yaml

```diff
@@ -12,6 +12,33 @@ spec:
   restartPolicy: Always
   hostNetwork: true
   containers:
+  - name: cluster-policy-controller
+    image: quay.io/openshift/origin-cluster-policy-controller:v4.3
+    imagePullPolicy: {{ .ImagePullPolicy }}
+    ports:
+      - containerPort: 10357
+    command: ["cluster-policy-controller", "start"]
+    args:
+    - --config=/etc/kubernetes/config/{{ .ConfigFileName }}
+    - --kubeconfig=/etc/kubernetes/secrets/kubeconfig
+    - --alsologtostderr
```

Comment:
@sanchezl for the command and args, can you confirm what's required here?
When I run `./cluster-policy-controller start`:
```
$ ./cluster-policy-controller start --help
Start the Cluster Policy Controller

Usage:
  cluster-policy-controller start [flags]

Flags:
      --config string   Location of the master configuration file to run from.
  -h, --help            help for start

Global Flags:
      --alsologtostderr                  log to standard error as well as files
---
```
does this controller need the `config` (and/or `kubeconfig`)? or is that leftover from openshift-controller-manager cmd.  thanks for your help.

Reply:
looking at the pod the PR test cluster, I don't see this container in the static pod.. maybe it needs to be added here instead? 
https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/bindata/v4.1.0/kube-controller-manager/pod.yaml

Reply:
Looks like it still needs the config.
I created pull request to remove references to the part of the config no longer needed: https://github.com/openshift/cluster-policy-controller/pull/5.

Reply:
cool thanks!

Reply:
I opened a PR to make the config optional (we were using an empty config file anyway).
https://github.com/openshift/cluster-policy-controller/pull/6

Reply:
got it, thanks